Beilagen/Salate/RheinischerKartoffelsalat.md

```diff
@@ -0,0 +1,32 @@
+# Rheinischer Kartoffelsalat
+
+Der Rheinische Kartoffelsalat schmeckt einfach! Er zählt zu den beliebtesten Klassikern seiner Art und kommt so richtig schön deftig daher. Lecker!
+
+## Zutaten
+
+* 1,5 kg festkochende Kartoffeln
+* Salz
+* 1/2 Kranz Fleichwurst (ca. 300 g)
+* 8-10 hartgekochte Eier
+* 3-4 Gewürzgurken
+* 1 Ei
+* 1 TL Essig
+* 1/2 TL Senf
+* 1 EL Zucker
+* ca. 200 ml Sonnenblumenöl
+* 3 EL Salatcreme, Pfeffer
+* 1/2 Bund Schnittlauch
+
+## Anleitung
+
+1. Kartoffeln waschen und in Salzwasser ca. 20 Minuten kochen. Kartoffeln abgießen, etwas abkühlen lassen und pellen. Wurst und gekochte Eier pellen. Gurken abtropfen lassen. Kartoffeln, Fleischwurst, Gurken und Eier in ca. 1 1⁄2 cm große Würfel schneiden. Alles in eine Schüssel geben.
+2. Für das Dressing das übrige Ei, Essig, Senf und Zucker gründlich verquirlen und das Sonnenblumenöl nach und nach in einem dünnen Strahl kräftig unterschlagen. Mit der Salatcreme verrühren, mit Salz und Pfeffer abschmecken.
+3. Dressing unter den Salat heben und ca. 60 Minuten ziehen lassen. Schnittlauch waschen, trocken schütteln und in Röllchen schneiden. Über den Salat streuen.
```

Comment:
Ich würde mir wünschen, dass die einzelnen Schritte noch in kleinere Schritte gepackt werden. Zum Beispiel:
1. Kartoffeln waschen und in Salzwasser ca. 20 Minuten kochen.
2. Kartoffeln abgießen, etwas abkühlen lassen und pellen. 
3. Wurst und gekochte Eier pellen. Gurken abtropfen lassen. Kartoffeln, Fleischwurst, Gurken und Eier in ca. 1 1⁄2 cm große Würfel schneiden. Alles in eine Schüssel geben.